module/VuFind/src/VuFind/AjaxHandler/GetResultCount.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Ajax Controller for Libraries Extension
```

Comment:
I think it would probably make the most sense to change this to match the class comment below (`"Get Result Counts" AJAX Handler`).